src/get-started/light-client.md

```diff
@@ -35,272 +35,216 @@ service vary depending on the business logic of each service.
+
+### Specify Transaction Payload
 
-Generate a signing key pair for signing and sending the transaction. The public
-key forms a separate field in the transaction header.
+Prepare the transaction payload according to the above-defined schema:
 
 ```javascript
-const keyPair = Exonum.keyPair()
+// Assume keys for both wallets are specified as follows:
+const alice = exonum.keyPair()
+const bob = exonum.keyPair()
+// Recall that we use addresses in the service, not public keys!
+const bobAddress = exonum.publicKeyToAddress(bob.publicKey)
+
+const transferPayload = {
+  to: { data: exonum.hexadecimalToUint8Array(bobAddress) },
+  amount: 50,
+  seed: exonum.randomUint64()
+}
 ```
 
-!!! note
-    In our Timestamping Service we generate a new signing key pair for each new
-    timestamp. On the contrary, in the Cryptocurrency Advanced Service we
-    generate only
-    one key pair that corresponds to a certain wallet and its user and, thus, is
-    applied for signing all transactions made on its behalf.
+`publicKeyToAddress` function provides a handy way to convert
+a hex-encoded public key to
+an [address](../advanced/service-interaction.md#authorization-via-services).
 
-#### Define a Transaction Schema
+### Sign Transaction
 
-Define `CreateTimestamp` transaction schema and its field types:
+Now, the transaction can be signed using a static method in `Transfer`:
 
 ```javascript
-const CreateTimestamp = Exonum.newTransaction({
-   author: keyPair.publicKey,
-   service_id: 130,
-   message_id: 0,
-   schema: timestamping.TxTimestamp
-})
+const transaction = Transfer.create(transferPayload, alice)
 ```
 
-- `author` is author’s public key
-- `service_id` represents the identifier of the service. Check the identifier
-  in the source code of the service (the smart-contract designed in Rust or
-  Java)
-- `message_id` represents the identifier of the transaction type in the
-  service. Corresponds to the index number of the transaction in the service
-  source code. Enumeration starts with `0`
-- `schema` schema of the `TxTimestamp` type defined above.
-
-#### Define Transaction Data
+This will create a transaction object conforming to the following
+generic TypeScript interface:
+
+```typescript
+interface Verified<T> {
+  // Schema of the payload defined as a `protobufjs` type.
+  schema: protobufjs.Type;
+  // Parametric payload.
+  payload: T;
+  // Hex-encoded Ed25519 public key (64 hex digits).
+  author: string;
+  // Hex-encoded Ed25519 signature (128 hex digits).
+  signature: string;
+  // Serialized Protobuf message in the format defined by Exonum.
+  bytes: Uint8Array;
+
+  // Returns serialized message.
+  serialize(): Uint8Array;
+  // Returns hex-encoded SHA-256 digest of the serialized message.
+  hash(): string;
+}
+```
 
-Prepare transaction data according to the above-defined schema:
+In case of `Transfer`, type param `T` is the transaction payload
```

Comment:
```suggestion
In case of `Transfer`, type param of `T` is the transaction payload
```

Reply:
Eh, `T` is the name of the type param; i.e., "type param" acts as a qualifier.

Reply:
Yeah. Sure.